nym-vpn/ui/src/pages/home/HopSelect.tsx

```diff
@@ -0,0 +1,43 @@
+import { useTranslation } from 'react-i18next';
+import { NodeConfig } from '../../types';
+interface HopSelectProps {
+  country: NodeConfig;
+  onClick: () => void;
+  node: 'entry' | 'exit';
+}
+export default function HopSelect({ node, country, onClick }: HopSelectProps) {
```

Comment:
suggestion: add blank lines between the import statements block, types definition block and function component definition
(better code readability)